### DISCUSSION TITLE
Model won't run - compiler issues?

### DISCUSSION THREAD

**Discussion by benne94** [phydrus/phydrus#59] [Compiling Fortran Code] [answered]
Hi,

I'm new to Hydrus / PHydrus and have been trying to follow the instructions on the previous post about compiling the Hydrus.exe using MinGW.  I followed every step to the letter and ran the make.bat file after moving into the source folder so that my "source_code/source" folder now contains the "make.bat" and "makefile" files.  I tried running "make" from cmd before moving into the "source" subfolder and it couldnt find the various ".FOR" files.

Is there any more advice on how to compile the .exe file, I'd love to use the PHydrus can't get the .exe set up!

FYI it writes the input files then I get the error message 

WARNING: Hydrus-1D Simulation Unsuccessful.
CompletedProcess(args=['hydrus.exe', '02_Hydrus\\01_Output', '-1'], returncode=3221225781)

I also tried with the downloadable v4.08.exe available online and had another error message for [PCPINFO.SYS] being erroneous.

**Comment by benne94** [comment, marked as answer]
I managed to get it to run by moving the .exe into the ws folder

ws = r"06_UnsatModelling\02_Hydrus"
#path to hydrus 1-d executable
exe = "hydrus.exe"r

**Comment by raoulcollenteur** [comment]
Moved this thread to Discussions and tagged with "compiling issues" to help others. Happy you got it working!

Thanks, Raoul
